notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -199,8 +199,8 @@
             :items="noteAllChildren"
             item-key="noteId">
             <template #label="{ item }">
-              <v-list-item-title @click="openNoteChild(item)" class="body-2 clickable primary--text">
-                <span>{{ item.name }}</span>
+              <v-list-item-title class="body-2 content-link clickable primary--text">
+                <a :href="getNoteLink(item.noteId)"><span>{{ item.name }}</span></a>
```

Comment:
For a common UX with other parts of the product, can you use please `<v-list-item :href` instead of `<v-list-item-title....<a :href`

```suggestion
              <v-list-item :href="getNoteLink(item.noteId)" class="body-2 primary--text">
                {{ item.name }}
              </v-list-item>
```

Besides, to not use a method here and use a computed instead, can you add a dedicated component for Tree leaf.

Reply:
done and to check